include/NovelRT/Maths/GeoVector2F.h

```diff
@@ -21,157 +24,789 @@ namespace NovelRT::Maths
+         *      4\\
+         *      5
+         *      \end{pmatrix}
+         *      =
+         *      \begin{pmatrix}
+         *      2 - 4\\
+         *      3 - 7
+         *      \end{pmatrix}
+         *      \rightarrow
+         *      \begin{pmatrix}
+         *      -2\\
+         *      -4
+         *      \end{pmatrix}
+         * \f]
+         *
+         * @param other The other vector who's components get subtracted from this vector.
+         * @return A GeoVecor2F who's components consist of the entrywise difference of this and another GeoVector2F.
+         */
         inline GeoVector2F operator-(GeoVector2F other) const noexcept
         {
             return GeoVector2F(*reinterpret_cast<const glm::vec2*>(this) - *reinterpret_cast<const glm::vec2*>(&other));
         }
 
+        /**
+         * @brief Constructs a GeoVector2F containing the Hadamard product of this and another GeoVector2F.
+         *
+         * @details
+         * An example of a Hadamard product of two vectors: \f[
+         *      \begin{pmatrix}
+         *      2\\
+         *      3
+         *      \end{pmatrix}
+         *      \circ
+         *      \begin{pmatrix}
+         *      4\\
+         *      5
+         *      \end{pmatrix}
+         *      =
+         *      \begin{pmatrix}
+         *      2 * 4\\
+         *      3 * 5
+         *      \end{pmatrix}
+         *      \rightarrow
+         *      \begin{pmatrix}
+         *      8\\
+         *      15
+         *      \end{pmatrix}
+         * \f]
+         * @param other The other vector who's components get multiplied by this vector's components.
+         * @return A GeoVecor2F who's components consist of the Hadamard product of this and another GeoVector2F.
+         */
         inline GeoVector2F operator*(GeoVector2F other) const noexcept
         {
             return GeoVector2F(*reinterpret_cast<const glm::vec2*>(this) * *reinterpret_cast<const glm::vec2*>(&other));
         }
 
+        /**
+         * @brief Constructs a GeoVector2F containing the quotents of a Hadamard division using this and another
+         * GeoVector2F.
+         *
+         * @details
+         * An example of a Hadamard division of two vectors: \f[
+         *      \begin{pmatrix}
+         *      8\\
+         *      6
+         *      \end{pmatrix}
+         *      \oslash
+         *      \begin{pmatrix}
+         *      2\\
+         *      3
+         *      \end{pmatrix}
+         *      =
+         *      \begin{pmatrix}
+         *      \tfrac{8}{2}\\
+         *      \tfrac{6}{3}
+         *      \end{pmatrix}
+         *      \rightarrow
+         *      \begin{pmatrix}
+         *      4\\
+         *      2
+         *      \end{pmatrix}
+         * \f]
+         * @param other The other vector who's component act as a divisor for this vector's components.
+         * @return A GeoVecor2F who's components consist of the entrywise quotent of this and another GeoVector2F.
```

Comment:
```suggestion
         * @return A GeoVector2F where the components consist of the entrywise quotient of this and another GeoVector2F.
```

I think you meant "quotient" here.